config/gnu-warnings/developer-10

```diff
@@ -0,0 +1,6 @@
+# New warning
+-Warith-conversion
+
+# Enable static analysis of program flow
+-fanalyzer
+-fdiagnostics-path-format=none
```

Comment:
This is fine here for now, but I plan to add gcc sanitizer support and we may want to move these there when that functionality is available.